e2e/pagination/pagination.e2e-spec.ts

```diff
@@ -121,4 +150,28 @@ test.describe.parallel(`Pagination tests`, () => {
 		expectedState.pages = ['A', 'B', 'C', 'D', 'E(current)', 'F'];
 		await expect.poll(() => paginationState(paginationPO1)).toEqual(expectedState);
 	});
+
+	test(`Hrefs custom for pages`, async ({page}) => {
+		const stateWithCustomHrefs = {
+			rootClasses: ['au-pagination', 'pagination'],
+			disabled: null,
+			isPreviousDisabled: false,
+			isNextDisabled: false,
+			pages: ['1', '2', '3', '4(current)', '5', '6'],
+			isFirstDisabled: false,
+			isLastDisabled: false,
+			hrefs: [
+				'#/pagination/hash#1',
+				'#/pagination/hash#2',
+				'#/pagination/hash#3',
+				'#/pagination/hash#4',
+				'#/pagination/hash#5',
+				'#/pagination/hash#6',
+			],
+		};
+		const paginationPO = new PaginationPO(page, 0);
+		await page.goto('#/pagination/hash');
+		await paginationPO.locatorRoot.waitFor();
+		await expect.poll(() => paginationState(paginationPO)).toEqual(stateWithCustomHrefs);
```

Comment:
Maybe we can test more, by including the click on one page and check the state (it will make sense if the boundaries href are included), and click on the browser history go back (I think there is a specific instruction with Playwright)

Reply:
indeed. nice point for the browser history.